lambdas/db-migration/main.tf

```diff
@@ -0,0 +1,144 @@
+# Input variables
+
+variable "permissions_boundary_arn" {
+  type = string
+}
+
+variable "prefix" {
+  type = string
+}
+
+variable "pg_host" {
+  type = string
+}
+
+variable "pg_user" {
+  type = string
+}
+
+variable "pg_password" {
+  type = string
+}
+
+variable "subnet_ids" {
+  type    = list(string)
+  default = []
+}
+
+variable "tags" {
+  type    = map(string)
+  default = {}
+}
+
+variable "vpc_id" {
+  type    = string
+  default = null
+}
+
+# Local variables
```

Comment:
We've gone around this horn a few times: but I would prefer the consistency of having variables in variables.tf, outputs in outputs.tf, etc. I feel like the nice broken up files are a little easier to navigate especially with consistent naming

Reply:
dc4009f55